### PR TITLE
docs: Add stability to README output

### LINKED ISSUE #62: Stabilize rendering of README
We need an output hook (?) that normalizes all `0x.....` strings. Also need to use only queries with stable sort order, or set the `DUCKPLYR_OUTPUT_ORDER` env var to `TRUE`.

### PR DESCRIPTION
**Pull request by maelle**
Fix #62

However some sort of order might still be needed? I see other lines in the diff, however if I knit again I don't get any difference.

### CODE REVIEW

File: README.Rmd [hunk 1]
```diff
@@ -15,6 +15,16 @@ knitr::opts_chunk$set(
 pkgload::load_all()
 
 set.seed(20230702)
+
+clean_output <- function(x, options) {
+  gsub("0x[0-9a-f]+", "0xdeadbeef", x)
```
Comment (maelle) [inline README.Rmd:20]: I used the same function as in https://github.com/duckdblabs/duckplyr/blob/13e5d48551982d140bff36de646c7dfdf3602532/tests/testthat/test-relational-duckdb.R#L99

I had never heard of 0xdeadbeef before.

**Comment by maelle** [comment]
@krlmlr is this a tibble option we need to set in README.Rmd?

**Comment by krlmlr** [comment]
I tried rendering with vanilla, same output.

Would it be possible to make the box look like a box again? Happy to merge then.

**Comment by maelle** [comment]
It's now stable... on my machine :sweat_smile: And it looks like a box again! :beverage_box:

**Pull request merged**